tsconfig.json

```diff
@@ -18,6 +18,6 @@
     "typeRoots": ["node_modules/@types"],
     "esModuleInterop": true
   },
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "tests/**/*"],
```

Comment:
now we include tests/** in build/dist? Why is it changed?

Reply:
I think I can remove it. It was done by @bogdoslavik because he has issues with IDE. After this we have:
```
"exclude": ["**/*.test.ts"]
```
So, it would add only test utils to build, though we don't need them.
Fixed